src/it/scala/com/tuplejump/kafka/connect/cassandra/CassandraSinkTaskSpec.scala

```diff
@@ -67,4 +70,73 @@ class CassandraSinkTaskSpec extends AbstractFlatSpec {
     rowCount should be(2)
   }
 
+
+  it should "save records in cassandra with custom field mapping" in {
+    val topicName = "test_fieldmap_topic"
+    val tableName = "test.fieldmap"
+    val config = sinkConfig((topicName, tableName))
+
+    val sinkTask = new CassandraSinkTask()
+    val mockContext = mock[SinkTaskContext]
+
+    val fieldMapping: JSONObject = JSONObject(Map(
```

Comment:
`CassandraSinkTask.start` takes a Properties JMap[String, String] so we are adding the property "field.mapping" with the stringified JSON as the value.